public/blog/announcing-full-portable-solidity-debugger.md

```diff
@@ -68,7 +68,7 @@ $ npm install -g truffle
 
 You may have to use `sudo` with the above commands if you're running Linux.
 
-More information about how to use Truffle can be found in [our documentation](/docs). Feel free to reach out on [Gitter](https://gitter.im/consensys/truffle) with any questions.
+More information about how to use Truffle can be found in [our documentation](/docs), and feel free to reach out on [Gitter](https://gitter.im/consensys/truffle) for community help from fellow Trufflers.
```

Comment:
I'd make this more "you focused". e.g., 

> and feel free to reach out on Gitter for help from your fellow Trufflers. 

To me, "fellow Trufflers" and "community" are synonymous, making one of them redundant. I'd choose "fellow Trufflers" to community because it feels more like a community that way.